tardis/io/atom_data/base.py

```diff
@@ -164,23 +164,33 @@ def from_hdf(cls, fname=None):
                 try:
                     dataframes[name] = store[name]
                 except KeyError:
+                    logger.debug("\n\tDataframe does not contain NAME column")
                     nonavailable.append(name)
 
             atom_data = cls(**dataframes)
 
             try:
                 atom_data.uuid1 = store.root._v_attrs["uuid1"].decode("ascii")
             except KeyError:
+                logger.debug(
+                    "\n\tUUID not available for Atom Data\n\tSetting value to NONE"
```

Comment:
I think NONE should be written as None to make it clear that it is the Python object None.